services/src/handlers/wms.rs

```diff
@@ -441,10 +433,11 @@ mod tests {
     async fn test_get_capabilities() {
         let res = get_capabilities_test_helper(Method::GET).await;
 
-        assert_eq!(res.status(), 200);
+        // assert_eq!(res.status(), 200);
```

Comment:
Why did you remove this?

Reply:
for seeing the output. I forgot to revert it back. b7ebd05ac4a1b0120193696140d6b238ee129fa0